ebt/Dialog/BtReciever.vala

```diff
@@ -20,7 +20,7 @@
  *
```

Comment:
You haven't changed the name of the file to the correct spelling.

Reply:
Ok